README.md

```diff
@@ -539,6 +543,96 @@ To avoid or work around caching, you can do the following:
 - Options.
 - Asynchronous by default (though can be run synchronously).
 
+## Usage for end users
+
+When configuring a tool, you can use multiple file formats and put these in multiple places.
+
+Usually, a tool would mention this in its own README file,
+but by default, these are the following places, where the `moduleName` variable represents the name of the tool:
+
+```js
+[
+  'package.json',
+  `.${moduleName}rc`,
+  `.${moduleName}rc.json`,
+  `.${moduleName}rc.yaml`,
+  `.${moduleName}rc.yml`,
+  `.${moduleName}rc.js`,
+  `.${moduleName}rc.cjs`,
+  `.config/${moduleName}rc`,
+  `.config/${moduleName}rc.json`,
+  `.config/${moduleName}rc.yaml`,
+  `.config/${moduleName}rc.yml`,
+  `.config/${moduleName}rc.js`,
+  `.config/${moduleName}rc.cjs`,
+  `${moduleName}.config.js`,
+  `${moduleName}.config.cjs`,
+]
```

Comment:
Perhaps this would be clearer if in the same format as the `searchPlaces` config:
```
package.json
.{name}rc
.{name}rc.json
...
```

Reply:
`searchPlaces` actually uses the same format as this, but I made it a bit clearer anyway since this is end user documentation.